internal/drivers/utils.go

```diff
@@ -19,6 +19,27 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+func checkExpose(exposeERIs []string, rdmaDevice string) bool {
+	if len(exposeERIs) == 1 && exposeERIs[0] == "" {
+		return true
+	}
+	for _, dev := range exposeERIs {
+		if dev == rdmaDevice {
+			return true
+		}
+	}
+	return false
+}
+
+func isErdmaNetworkInterface(macAddress string, node_guid string) bool {
```

Comment:
existed `GetERdmaFromLink` function can judge whether the network interface is erdma.
https://github.com/AliyunContainerService/alibabacloud-erdma-controller/blob/7cccef1a4f40767bd564ed92e5ed28c77b3561e0/internal/drivers/utils.go#L114C6-L114C22